lnbits/wallets/opennode.py

```diff
@@ -119,14 +123,16 @@ async def get_payment_status(self, checking_id: str) -> PaymentStatus:
         if r.is_error:
             return PaymentStatus(None)
 
+        data = r.json()["data"]
         statuses = {
             "initial": None,
             "pending": None,
             "confirmed": True,
-            "error": False,
+            "error": None,
```

Comment:
Why `None` here and not `False`?

Reply:
Good question, I did not have to change that. It makes no difference for invoices to be `False` or `None`, I suppose I wanted to be sure that is' an invoice-related error ("failed") instead of some system response "error". Will revert to be sure.

Reply:
Nevermind, I think it makes sense.